src/control/cmd/dmg/pool.go

```diff
@@ -42,8 +42,17 @@ const (
 	msgSizeNoNumber = "size string doesn't specify a number"
 	msgSizeZeroScm  = "non-zero scm size is required"
 	maxNumSvcReps   = 13
+	mibsInMB        = 0.90949470177293 // convert base2->base10
```

Comment:
Rather than having a constant of a long float value, I'd suggest performing 10^6 / 2^20 directly in the conversion method. Makes it more obvious what is going on there.

Reply:
I did consider that so will change